.env.test.example

```
@@ -32,3 +32,9 @@ AWS_S3_BUCKET_PREFIX="bucket-prefix-name"
 # Callback Timeouts (in seconds)
 CALLBACK_CONNECT_TIMEOUT = 3
 CALLBACK_READ_TIMEOUT = 10
+
+OPENAI_API_KEY=""
+
+KAAPI_GUARDRAILS_AUTH=""
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)